src/crimson/os/seastore/linked_tree_node.h

```diff
@@ -967,37 +965,46 @@ class ChildNode : public BaseChildNode<ParentT, key_t> {
 protected:
   void take_prior_parent_tracker() {
```

Comment:
Is it better to:
1. merge `take_prior_parent_tracker()` and `adjust_parent()`
2. move to the private section

?

Looks there is nothing to reuse.